neuron-model/tutorial-replicate-paper-figure/README.md

```diff
@@ -9,57 +9,57 @@ or the [1.0 stable release](https://github.com/cengique/drosophila-aCC-L3-motone
 Run the simulation with:
```

Comment:
Ok, I see you moved the README completely to the tutorial folder. This is good for change tracking, but I'd like the top part of this information to stay under the `neuron-model/` folder. Create a new `README.md` in there and copy the part before the start of the tutorial, so people looking at the folder have some instructions to start with. Also, link to the two tutorials contained in the subfolders.

Reply:
Actually, not just copy, but _move_ that text to the new README.